scripts/run_presubmit_checks_test.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/run_presubmit_checks.py."""
+
+from __future__ import annotations
+
+import builtins
+import os
+import subprocess
+
+from core.tests import test_utils
+
+from scripts import run_backend_tests
+from scripts import run_frontend_tests
+from scripts import run_presubmit_checks
+from scripts.linters import pre_commit_linter
+
+
+class RunPresubmitChecksTests(test_utils.GenericTestBase):
+    """Unit tests for scripts/run_presubmit_checks.py."""
+
+    def setUp(self) -> None:
+        super(RunPresubmitChecksTests, self).setUp()
```

Comment:
I think this can just be `super().setUp()` now that we're on python 3

Reply:
Done!